experiments/kompanion/cmd/summary/options.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package summary
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	// flag names.
+	kubeconfigFlag       = "kubeconfig"
+	reportNamePrefixFlag = "report-prefix"
+
+	targetNamespacesFlag = "target-namespaces"
+	ignoreNamespacesFlag = "exclude-namespaces"
+
+	targetObjectsFlag = "target-objects"
+
+	workerRoutinesFlag = "worker-routines"
+)
```

Comment:
might be nice to make some of these common options; can be added as a follow on!

Reply:
+1